chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -323,7 +323,8 @@
 end
 
 #### Initialize rails gems/database
-execute "bundle install --without none" do
+execute "bundle install #{"--deployment" if rails_env == "production"} --path /home/#{username}/.bundle" do
```

Comment:
You're right, it installs everything otherwise...
So basically we also want to run "--without production" when in development.